code/datums/components/uplink.dm

```diff
@@ -128,6 +129,17 @@
 	if(istype(item, /obj/item/stack/telecrystal))
 		load_tc(user, item)
 
+/datum/component/uplink/proc/on_examine(datum/source, mob/user, list/examine_list)
+	SIGNAL_HANDLER
+
+	if(user != owner)
+		return
+	examine_list += span_warning("\The [parent] contains your hidden uplink.")
+	if(unlock_code)
+		examine_list += span_warning("The code to unlock it is [unlock_code].")
```

Comment:
This should be in the same line as the "this contains your uplink" IMO. Then bold the unlock code with `span_bold`